draft-ietf-mls-protocol.md

```diff
@@ -2229,14 +2229,33 @@ Proposals. It instructs group members to update their representation of the
 state of the group by applying the proposals and advancing the key schedule.
 
 Each proposal covered by the Commit is identified by a ProposalID value, which
-contains the hash of the MLSPlaintext in which the Proposal was sent, using the
-hash function from the group's ciphersuite.
+identifies the proposal to be applied by value or by reference.  Proposals
+supplied by value are included directly in the Commit object.  Proposals
+supplied by reference are specified by including the hash of the MLSPlaintext in
+which the Proposal was sent, using the hash function from the group's
+ciphersuite.  For proposals whose application depends on who sent them (e.g.,
```

Comment:
> For proposals whose application depends on who sent them

I think this is true for all proposals, because they might get rejected depending on the application's policy.